packages/core/src/config/config.ts

```diff
@@ -50,7 +50,9 @@ function getIgnoreFilePath(configFile?: string): string | undefined {
       ? path.join(path.dirname(configFile), IGNORE_FILE)
       : path.join(configFile, IGNORE_FILE);
   } else {
-    return typeof process === 'undefined' ? undefined : path.join(process.cwd(), IGNORE_FILE);
+    return (typeof process === 'undefined' || typeof process.cwd === 'undefined')
```

Comment:
Could you check if `isBrowser`  from `packages/core/src/env.ts` works for your case instead? I believe it might be much convenient.